content/SCALE/SCALETutorials/Apps/_index.md

```diff
@@ -268,19 +268,17 @@ The **Stable** train includes the apps in the **charts** repository.
 Non-Enterprise systems show the **stable** catalog of apps by default.
 The **Stable** catalog train has official applications that are pre-configured and only require a name during deployment.
 
-Enterprise applications display automatically for Enterprise-licensed systems, but community users can add these apps using the **Manage Catalogs** screen.
-App trains display based on the **Trains** settings on the **Edit Catalog** screen.
+Enterprise applications display automatically for Enterprise-licensed systems, but community users can add these apps using the **Edit Catalog** screen.
```

Comment:
```suggestion
Enterprise applications display automatically for Enterprise-licensed systems, but community users can add these apps using the **Train Settings** screen.
```

Reply:
Fixing this in the next sprint on the next ticket. Late UI change from the last build.